### DISCUSSION TITLE
6187 Killed Error

### DISCUSSION THREAD

**Discussion by Joanone** [neurodesk/neurodesk.github.io#115] [Q&A] [answered]
Dear NeuroDesk Experts,

I repeatedly got this error while running a command on neurodesk. I'm not sure if this is related to the software i'm using or neurodesk itself. 

In addition, i ran this command on a different PC few days back and it was executed without any errors.

Please what do you think?
![ae0c18ae8cb66d0fc00664018eedb2f](https://user-images.githubusercontent.com/83198624/151758187-eb40a855-4e97-4ce2-9621-5462ab68f219.png)

**Comment by stebo85** [comment]
Dear @Joanone - that is very interesting! Thank you for reporting this.

- what do you mean with "different PC"? Did you also run it in Neurodesk on the other computer? Did you run the same version of mrtrix?

**Reply by Joanone** [reply]
Yes, i ran it on Neurodesk but with a different computer which had lower specifications, this current one i'm using is higher. The version of mrtrix is also the same.

**Reply by stebo85** [reply]
ok, that helps a lot. 

How much RAM do both PCs have and which processor types?

**Reply by Joanone** [reply]
the first has 16GB RAM and the current one has 32GB.

**Reply by stebo85** [reply]
Ok, are both docker versions using WSL or is one of them managing resources itself? (Check in docker settings)

Are both windows computers?

What are the different processer types?

**Comment by civier** [comment]
Hello @Joanone @stebo85

Processes in Linux might be automatically killed (i.e. terminated) if they run out of memory. It might be possible to trace the system logs to find out what happened exactly:
https://askubuntu.com/questions/709336/how-to-find-out-why-process-was-killed-on-server
If it doesn't tell you too much, can you please:
1. Open a terminal in Neurodesk
2. run **dmesg > log.txt**
3. attach log.txt to this discussion

Thanks ahead,
Oren

**Reply by Joanone** [reply]
okay @civier , thanks! I tried running the command in step 2 to attach the log.txt file  but i got this rather:

dmesg: read kernel buffer failed: Operation not permitted.

I'll go through the link you shared in details and see if i can go round it. 

Thanks.

**Reply by civier** [reply]
@stebo85 @aswinnarayanan Do you have a clue why dmesg won't run on Docker on Windows, but I don't have an issue running it on Docker on Linux (Neurodesktop version 20220124)?
Is it really taking messages from host rather than from within the container?
https://stackoverflow.com/questions/41171349/why-is-a-shell-within-a-docker-container-showing-dmesg-content-from-the-host
If yes, is it related to permissions or just not supported on Windows?

**Reply by civier** [reply]
@stebo85 @aswinnarayanan Despite this issue being solved, I'd still be interested to know what is the best way to find out why a process on a Docker running on Windows was killed? So if you know how to display dmesg (or its equivalent), that would be helpful.

**Reply by stebo85** [reply]
I don't know what the best way is.

**Comment by Joanone** [comment]
Yes, both of them use WSL and are windows systems .
I have attached a screenshot of the specifications for the PC that gave the error. I can send the other one in a short while.[cid:0B27EDC5-4216-4C52-A32B-E48E26E4927D-L0-001]

Sent from my iPhone

On Jan 31, 2022, at 5:35 PM, Steffen Bollmann ***@***.***> wrote:

﻿

Ok, are both docker versions using WSL or is one of them managing resources itself? (Check in docker settings)

Are both windows computers?

What are the different processer types?

—
Reply to this email directly, view it on GitHub<https://github.com/NeuroDesk/neurodesk.github.io/discussions/104#discussioncomment-2080143>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AT2YFIDUSNDX2NI4AYCQJL3UYZJVBANCNFSM5NFRDR3Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by stebo85** [reply]
@Joanone,

can you run:
cat /proc/cpuinfo
cat /proc/meminfo

in both and send the output?

Thank you

**Comment by Joanone** [comment]
Sure, I attached the output in this doc file.

Thanks.


Sent from my iPhone

On Jan 31, 2022, at 5:51 PM, Steffen Bollmann ***@***.***> wrote:

﻿

@Joanone<https://github.com/Joanone>,

can you run:
cat /proc/cpuinfo
cat /proc/meminfo

in both and send the output?

Thank you

—
Reply to this email directly, view it on GitHub<https://github.com/NeuroDesk/neurodesk.github.io/discussions/104#discussioncomment-2080231>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AT2YFICHIXL7ACNIILWGUADUYZLSTANCNFSM5NFRDR3Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by stebo85** [reply]
Dear @Joanone 

The doc file is not coming through on github. Can you post the text?

**Comment by Joanone** [comment, marked as answer]
Okay.sure.




(base) ***@***.***:~$ cat /proc/cpuinfo

processor​: 0

vendor_id​: GenuineIntel

cpu family​: 6

model​​: 165

model name​: Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz

stepping​: 5

microcode​: 0xffffffff

cpu MHz​​: 3600.012

cache size​: 20480 KB

physical id​: 0

siblings​: 10

core id​​: 0

cpu cores​: 5

apicid​​: 0

initial apicid​: 0

fpu​​: yes

fpu_exception​: yes

cpuid level​: 21

wp​​: yes

flags​​: fpu vme de pse tsc msr pae mce cx8 apic sepmtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss htsyscall nx pdpe1gb rdtscp lm constant_tsc rep_good noplxtopology cpuid pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibpibrs_enhanced fsgsbase bmi1 avx2 smep bmi2 erms invpcidrdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsavesflush_l1d arch_capabilities

bugs​​: spectre_v1 spectre_v2 spec_store_bypassswapgs itlb_multihit

bogomips​: 7200.02

clflush size​: 64

cache_alignment​: 64

address sizes​: 39 bits physical, 48 bits virtual

power management:



processor​: 1

vendor_id​: GenuineIntel

cpu family​: 6

model​​: 165

model name​: Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz

stepping​: 5

microcode​: 0xffffffff

cpu MHz​​: 3600.012

cache size​: 20480 KB

physical id​: 0

siblings​: 10

core id​​: 0

cpu cores​: 5

apicid​​: 1

initial apicid​: 1

fpu​​: yes

fpu_exception​: yes

cpuid level​: 21

wp​​: yes

flags​​: fpu vme de pse tsc msr pae mce cx8 apic sepmtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss htsyscall nx pdpe1gb rdtscp lm constant_tsc rep_good noplxtopology cpuid pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibpibrs_enhanced fsgsbase bmi1 avx2 smep bmi2 erms invpcidrdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsavesflush_l1d arch_capabilities

bugs​​: spectre_v1 spectre_v2 spec_store_bypassswapgs itlb_multihit

bogomips​: 7200.02

clflush size​: 64

cache_alignment​: 64

address sizes​: 39 bits physical, 48 bits virtual

power management:



processor​: 2

vendor_id​: GenuineIntel

cpu family​: 6

model​​: 165

model name​: Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz

stepping​: 5

microcode​: 0xffffffff

cpu MHz​​: 3600.012

cache size​: 20480 KB

physical id​: 0

siblings​: 10

core id​​: 1

cpu cores​: 5

apicid​​: 2

initial apicid​: 2

fpu​​: yes

fpu_exception​: yes

cpuid level​: 21

wp​​: yes

flags​​: fpu vme de pse tsc msr pae mce cx8 apic sepmtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss htsyscall nx pdpe1gb rdtscp lm constant_tsc rep_good noplxtopology cpuid pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibpibrs_enhanced fsgsbase bmi1 avx2 smep bmi2 erms invpcidrdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsavesflush_l1d arch_capabilities

bugs​​: spectre_v1 spectre_v2 spec_store_bypassswapgs itlb_multihit

bogomips​: 7200.02

clflush size​: 64

cache_alignment​: 64

address sizes​: 39 bits physical, 48 bits virtual

power management:



processor​: 3

vendor_id​: GenuineIntel

cpu family​: 6

model​​: 165

model name​: Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz

stepping​: 5

microcode​: 0xffffffff

cpu MHz​​: 3600.012

cache size​: 20480 KB

physical id​: 0

siblings​: 10

core id​​: 1

cpu cores​: 5

apicid​​: 3

initial apicid​: 3

fpu​​: yes

fpu_exception​: yes

cpuid level​: 21

wp​​: yes

flags​​: fpu vme de pse tsc msr pae mce cx8 apic sepmtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss htsyscall nx pdpe1gb rdtscp lm constant_tsc rep_good noplxtopology cpuid pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibpibrs_enhanced fsgsbase bmi1 avx2 smep bmi2 erms invpcidrdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsavesflush_l1d arch_capabilities

bugs​​: spectre_v1 spectre_v2 spec_store_bypassswapgs itlb_multihit

bogomips​: 7200.02

clflush size​: 64

cache_alignment​: 64

address sizes​: 39 bits physical, 48 bits virtual

power management:



processor​: 4

vendor_id​: GenuineIntel

cpu family​: 6

model​​: 165

model name​: Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz

stepping​: 5

microcode​: 0xffffffff

cpu MHz​​: 3600.012

cache size​: 20480 KB

physical id​: 0

siblings​: 10

core id​​: 2

cpu cores​: 5

apicid​​: 4

initial apicid​: 4

fpu​​: yes

fpu_exception​: yes

cpuid level​: 21

wp​​: yes

flags​​: fpu vme de pse tsc msr pae mce cx8 apic sepmtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss htsyscall nx pdpe1gb rdtscp lm constant_tsc rep_good noplxtopology cpuid pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibpibrs_enhanced fsgsbase bmi1 avx2 smep bmi2 erms invpcidrdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsavesflush_l1d arch_capabilities

bugs​​: spectre_v1 spectre_v2 spec_store_bypassswapgs itlb_multihit

bogomips​: 7200.02

clflush size​: 64

cache_alignment​: 64

address sizes​: 39 bits physical, 48 bits virtual

power management:



processor​: 5

vendor_id​: GenuineIntel

cpu family​: 6

model​​: 165

model name​: Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz

stepping​: 5

microcode​: 0xffffffff

cpu MHz​​: 3600.012

cache size​: 20480 KB

physical id​: 0

siblings​: 10

core id​​: 2

cpu cores​: 5

apicid​​: 5

initial apicid​: 5

fpu​​: yes

fpu_exception​: yes

cpuid level​: 21

wp​​: yes

flags​​: fpu vme de pse tsc msr pae mce cx8 apic sepmtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss htsyscall nx pdpe1gb rdtscp lm constant_tsc rep_good noplxtopology cpuid pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibpibrs_enhanced fsgsbase bmi1 avx2 smep bmi2 erms invpcidrdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsavesflush_l1d arch_capabilities

bugs​​: spectre_v1 spectre_v2 spec_store_bypassswapgs itlb_multihit

bogomips​: 7200.02

clflush size​: 64

cache_alignment​: 64

address sizes​: 39 bits physical, 48 bits virtual

power management:



processor​: 6

vendor_id​: GenuineIntel

cpu family​: 6

model​​: 165

model name​: Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz

stepping​: 5

microcode​: 0xffffffff

cpu MHz​​: 3600.012

cache size​: 20480 KB

physical id​: 0

siblings​: 10

core id​​: 3

cpu cores​: 5

apicid​​: 6

initial apicid​: 6

fpu​​: yes

fpu_exception​: yes

cpuid level​: 21

wp​​: yes

flags​​: fpu vme de pse tsc msr pae mce cx8 apic sepmtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss htsyscall nx pdpe1gb rdtscp lm constant_tsc rep_good noplxtopology cpuid pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibpibrs_enhanced fsgsbase bmi1 avx2 smep bmi2 erms invpcidrdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsavesflush_l1d arch_capabilities

bugs​​: spectre_v1 spectre_v2 spec_store_bypassswapgs itlb_multihit

bogomips​: 7200.02

clflush size​: 64

cache_alignment​: 64

address sizes​: 39 bits physical, 48 bits virtual

power management:



processor​: 7

vendor_id​: GenuineIntel

cpu family​: 6

model​​: 165

model name​: Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz

stepping​: 5

microcode​: 0xffffffff

cpu MHz​​: 3600.012

cache size​: 20480 KB

physical id​: 0

siblings​: 10

core id​​: 3

cpu cores​: 5

apicid​​: 7

initial apicid​: 7

fpu​​: yes

fpu_exception​: yes

cpuid level​: 21

wp​​: yes

flags​​: fpu vme de pse tsc msr pae mce cx8 apic sepmtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss htsyscall nx pdpe1gb rdtscp lm constant_tsc rep_good noplxtopology cpuid pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibpibrs_enhanced fsgsbase bmi1 avx2 smep bmi2 erms invpcidrdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsavesflush_l1d arch_capabilities

bugs​​: spectre_v1 spectre_v2 spec_store_bypassswapgs itlb_multihit

bogomips​: 7200.02

clflush size​: 64

cache_alignment​: 64

address sizes​: 39 bits physical, 48 bits virtual

power management:



processor​: 8

vendor_id​: GenuineIntel

cpu family​: 6

model​​: 165

model name​: Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz

stepping​: 5

microcode​: 0xffffffff

cpu MHz​​: 3600.012

cache size​: 20480 KB

physical id​: 0

siblings​: 10

core id​​: 4

cpu cores​: 5

apicid​​: 8

initial apicid​: 8

fpu​​: yes

fpu_exception​: yes

cpuid level​: 21

wp​​: yes

flags​​: fpu vme de pse tsc msr pae mce cx8 apic sepmtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss htsyscall nx pdpe1gb rdtscp lm constant_tsc rep_good noplxtopology cpuid pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibpibrs_enhanced fsgsbase bmi1 avx2 smep bmi2 erms invpcidrdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsavesflush_l1d arch_capabilities

bugs​​: spectre_v1 spectre_v2 spec_store_bypassswapgs itlb_multihit

bogomips​: 7200.02

clflush size​: 64

cache_alignment​: 64

address sizes​: 39 bits physical, 48 bits virtual

power management:



processor​: 9

vendor_id​: GenuineIntel

cpu family​: 6

model​​: 165

model name​: Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz

stepping​: 5

microcode​: 0xffffffff

cpu MHz​​: 3600.012

cache size​: 20480 KB

physical id​: 0

siblings​: 10

core id​​: 4

cpu cores​: 5

apicid​​: 9

initial apicid​: 9

fpu​​: yes

fpu_exception​: yes

cpuid level​: 21

wp​​: yes

flags​​: fpu vme de pse tsc msr pae mce cx8 apic sepmtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss htsyscall nx pdpe1gb rdtscp lm constant_tsc rep_good noplxtopology cpuid pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibpibrs_enhanced fsgsbase bmi1 avx2 smep bmi2 erms invpcidrdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsavesflush_l1d arch_capabilities

bugs​​: spectre_v1 spectre_v2 spec_store_bypassswapgs itlb_multihit

bogomips​: 7200.02

clflush size​: 64

cache_alignment​: 64

address sizes​: 39 bits physical, 48 bits virtual



















(base) ***@***.***:~$ cat /proc/meminfo

MemTotal:        2031364 kB

MemFree:          483724 kB

MemAvailable:    1006188 kB

Buffers:           28960 kB

Cached:           607384 kB

SwapCached:        32332 kB

Active:           290604 kB

Inactive:        1113112 kB

Active(anon):      33748 kB

Inactive(anon):   737396 kB

Active(file):     256856 kB

Inactive(file):   375716 kB

Unevictable:           0 kB

Mlocked:               0 kB

SwapTotal:       1048572 kB

SwapFree:         957836 kB

Dirty:                36 kB

Writeback:             0 kB

AnonPages:        740560 kB

Mapped:           220840 kB

Shmem:              7524 kB

KReclaimable:      43264 kB

Slab:              72660 kB

SReclaimable:      43264 kB

SUnreclaim:        29396 kB

KernelStack:       12288 kB

PageTables:         8976 kB

NFS_Unstable:          0 kB

Bounce:                0 kB

WritebackTmp:          0 kB

CommitLimit:     2064252 kB

Committed_AS:    4561092 kB

VmallocTotal:   34359738367 kB

VmallocUsed:       19000 kB

VmallocChunk:          0 kB

Percpu:             7104 kB

AnonHugePages:         0 kB

ShmemHugePages:        0 kB

ShmemPmdMapped:        0 kB

FileHugePages:         0 kB

FilePmdMapped:         0 kB

HugePages_Total:       0

HugePages_Free:        0

HugePages_Rsvd:        0

HugePages_Surp:        0

Hugepagesize:       2048 kB

Hugetlb:               0 kB

DirectMap4k:       79152 kB

DirectMap2M:     2017280 kB

Sent from my iPhone

On Jan 31, 2022, at 6:33 PM, Steffen Bollmann ***@***.***> wrote:

﻿

Dear @Joanone<https://github.com/Joanone>

The doc file is not coming through on github. Can you post the text?

—
Reply to this email directly, view it on GitHub<https://github.com/NeuroDesk/neurodesk.github.io/discussions/104#discussioncomment-2080512>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AT2YFICGTGOAFA5H66MHCGTUYZQPDANCNFSM5NFRDR3Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by stebo85** [reply]
@Joanone 

 This one (I assume it's the PC where you get the error) only has 2GB of memory available for the docker container, which is not enough. Can you check in the docker settings how much memory you provide?

**Comment by Joanone** [comment]
Okay. Thanks a lot for this! I would double check this again when I can and send the response.

Cheers.
Joan

Sent from my iPhone

On Jan 31, 2022, at 6:52 PM, Steffen Bollmann ***@***.***> wrote:

﻿

@Joanone<https://github.com/Joanone>

This one (I assume it's the PC where you get the error) only has 2GB of memory available for the docker container, which is not enough. Can you check in the docker settings how much memory you provide?

—
Reply to this email directly, view it on GitHub<https://github.com/NeuroDesk/neurodesk.github.io/discussions/104#discussioncomment-2080626>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AT2YFIHMCI6ZU5HH5D7OJF3UYZSX3ANCNFSM5NFRDR3Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by stebo85** [comment]
Ok,providing more memory to the desktop container should solve the problem
:)

**Reply by Joanone** [reply]
Absolutely right! @stebo85. That solved it. Thanks everyone.
Kind regards.

**Reply by stebo85** [reply]
Ok, great to hear :)

**Reply by civier** [reply]
@Joanone Many thanks for helping us with diagnosing this issue.
Might I ask if 2GB memory was the default when installing Docker for Windows, or whether it's a setting you chose at one point in the installation? Hopefully, we can use this information to develop some guidelines for users.

**Reply by Joanone** [reply]
@civier My pleasure. I suspect 2GB was the default, because throughout the installation, i don't remember checking the memory capacity for the docker container. I didn't know i had to also, i'm a bit new to using docker :).

But today, after i checked the settings in docker, and increased the memory from 2GB to about 50GB, everything went back to normal.

**Reply by civier** [reply]
Thank you @Joanone

@neuro-sevas (Stefanie, David White's student) Would you be willing to try and install Docker on your Windows 10 (following the instructions here: https://neurodesk.github.io/docs/neurodesktop/getting-started/windows/), and see whether 2GB is indeed the default setting? If this is so, maybe we should discuss on a recommended memory setting that would accomodate most Neuroimaging software, and include instructions on how to change it in the Windows installation page: https://neurodesk.github.io/docs/neurodesktop/getting-started/windows/
It shouldn't take much time to test it out, but will be very useful for future users

**Reply by neuro-sevas** [reply]
Hi @civier
Using the WSL2 backend I can't see memory stats in settings but running `docker info` in PowerShell returned a value of 12.43GB for memory (the PC has a total of 16GB RAM).

**Comment by civier** [comment]
Thanks a lot Stefanie for looking into it.

Do you mind running the Neurodesktop container on your Windows 10 and make
sure it is indeed has all of that memory allocated to it? (follow Steffen's
instructions earlier in this thread)
Maybe there is an allocation of memory per running container that is set to
a lower value than total memory available for docker.

Best,
Oren

**Reply by neuro-sevas** [reply]
No worries!

Here you go:
[cpuinfo.txt](https://github.com/NeuroDesk/neurodesk.github.io/files/7991220/cpuinfo.txt)
[meminfo.txt](https://github.com/NeuroDesk/neurodesk.github.io/files/7991221/meminfo.txt)

**Comment by civier** [comment]
Seems good. I now wonder why Joan had only 2G available? Maybe we wait to
see if other users encounter the same issue.

In the time being, Steff, maybe you can find our how to change the memory
settings of docker on Windows (in settings, I understand), and add it to
the FAQ section of NeuroDesk. Users usually have a process killed, so maybe
you can write a solution to the question "I got an error message 'XXX
Killed'". You should explain how to check for available memory and how to
increase it.
FAQ section is here:
https://neurodesk.github.io/docs/neurodesktop/troubleshooting/

Thanks ahead,
Oren
